flixel/tile/FlxTilemap.hx

```diff
@@ -290,14 +301,42 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 		
 		// Create debug tiles for rendering bounding boxes on demand
 		#if FLX_DEBUG
+		updateDebugTileBoundingBoxes();
+		updateDebugTileBoundingBoxesInit = true;
+		#end
+	}
+	
+	#if FLX_DEBUG
+	private function updateDebugTileBoundingBoxes():Void 
+	{
 		if (FlxG.renderBlit)
 		{
-			_debugTileNotSolid = makeDebugTile(FlxColor.BLUE);
-			_debugTilePartial = makeDebugTile(FlxColor.PINK);
-			_debugTileSolid = makeDebugTile(FlxColor.GREEN);
+			if (_debugTileSolid == null)
```

Comment:
too much repetitions.

Maybe it would be better have this?
```
private function updateDebugTileBoundingBoxes():Void 
	{
		if (FlxG.renderBlit)
		{
			_debugTileSolid = updateDebugTile(_debugTileSolid, debugColorScheme.solid);
			_debugTilePartial = updateDebugTile(_debugTilePartial, debugColorScheme.partial);
			_debugTileNotSolid = updateDebugTile(_debugTileNotSolid, debugColorScheme.notSolid);
		}
	}
	
	private function updateDebugTile(tileBitmap:BitmapData, color:FlxColor):BitmapData
	{
		if (tileBitmap != null && (tileBitmap.width != _tileWidth || tileBitmap.height != _tileHeight))
			tileBitmap = FlxDestroyUtil.dispose(tileBitmap);
		
		if (tileBitmap == null)
			tileBitmap = makeDebugTile(color);
		else
		{
			tileBitmap.fillRect(tileBitmap.rect, FlxColor.TRANSPARENT);
			drawDebugTile(tileBitmap, color);
		}
		
		return tileBitmap;
	}
```